pywr-schema/src/nodes/annual_virtual_storage.rs

```diff
@@ -53,24 +54,49 @@ impl AnnualVirtualStorageNode {
         tables: &LoadedTableCollection,
         data_path: Option<&Path>,
         inter_network_transfers: &[PywrMultiNetworkTransfer],
+        timeseries: &LoadedTimeseriesCollection,
     ) -> Result<(), SchemaError> {
         let cost = match &self.cost {
             Some(v) => v
-                .load(network, schema, domain, tables, data_path, inter_network_transfers)?
+                .load(
```

Comment:
These arguments are bit silly now. Not for this PR, but it would be a good idea to think about combining these into a struct at some point.